paasta_tools/setup_tron_namespace.py

```diff
@@ -187,16 +197,31 @@ def main():
                     for_validation=False,
                 )
                 ensure_service_accounts(job_configs)
+                if not args.bulk_config_fetch:
+                    if client.update_namespace(service, new_config):
+                        updated.append(service)
+                        log.debug(f"Updated {service}")
+                    else:
+                        skipped.append(service)
+                        log.debug(f"Skipped {service}")
 
-                if client.update_namespace(service, new_config):
-                    updated.append(service)
-                    log.debug(f"Updated {service}")
-                else:
-                    skipped.append(service)
-                    log.debug(f"Skipped {service}")
         except Exception:
-            log.exception(f"Update for {service} failed:")
+            log.exception(
+                f"Failed to create service account for {service} (will skip reconfiguring):"
```

Comment:
Should this change in error message + new_configs handling be moved to a gate on `args.bulk_config_fetch`, and the old error message/`failed` handling be used when we aren't using the flag?

Reply:
err, yea - that's a good catch!

Reply:
updated!